docs/basic-docs/atomicdex/atomicdex-api.md

```diff
@@ -1279,6 +1279,91 @@ curl --url "http://127.0.0.1:7783" --data "{\"userpass\":\"$userpass\",\"method\
 
 </div>
 
+## kmd\_rewards\_info
+
+**kmd_rewards_info**
+
+The `kmd_rewards_info` method returns rewards info of unspent outputs.
```

Comment:
Please add the note that this method only works when KMD coin is activated.

Reply:
Done